_posts/rfc/2015-06-10-message-api.md

```diff
@@ -29,37 +40,43 @@ Every **Thread** MUST provide the following information:
 * `unread_count`: the number of unread messages
 * `subject`: the thread subject. Empty string if none.
 * `contacts`: a list of links to contacts involved in the conversation
-* `importance_level`: the importance level associated to message
-* `privacy_index`: the privacy index associated to message
+* `importance_level`: the importance level associated to this conversation
+* `privacy_index`: the privacy index associated to this conversation
 * `messages`: a list of links to most recent messages. **<--- This is used for
-  the thread headline and for the detail view initial display**
+  the thread headline and for the detail view initial display**.
 * `attachments`: a list of links to most recent attachments.
-* `Tags`
+* `tags`: a list of tags associated to this conversation.
+
+## Attachment
+
+This resource concern attachment included in a **Message**.
+
+* `message_id`: Message reference this attachment is related to
+* `content_type`: MIME content type for this part
+* `data`: Part content
+
+## Message
 
 A **Message** can come from several providers (email, twitter, instant chat...).
 This is a common abstraction, every message MUST provide the following
 information:
 
-* `status`: read, draft, unread... **<-- Pleas provide some insight on the way
-  you think it should be handled**
+* `status`: draft, sending, sent, deleted for outcoming messages. unread, read, deleted 
+  for incoming ones.
 * `thread`: link to the thread the message is included in.
-* `sender`: link to contact who sent this message if any. **<--- should raw
-  information be provided otherwise? It should remain consistent**
-* `sent_at`: date and time the message was sent (most probably extracted from
-  headers or metadata, depending on the message source)
-* `received_at`: date and time the message was received on the CaliOpen platform
+* `sender`: link to contact who sent this message.
```

Comment:
the entity properties seems to be duplicated here and in recipients. I'm not sure about this for now